autoload/codi.vim

```diff
@@ -659,12 +660,19 @@ function! s:virtual_text_codi_handle_done(bufnr, output)
   call s:nvim_codi_output_to_virtual_text(a:bufnr, result)
 endfunction
 
+function! s:nvim_codi_clear_virtual_text()
+  let g:virtual_text_namespace = s:virtual_text_namespace
```

Comment:
Is this global variable used anywhere?

Reply:
Ups, a left that by accident. I used that for debugging. May I ask you, how do you echo or debug variables?

Because if I just echo something it does not show in the command output.

Reply:
You can set `g:codi#log` to a log file path and insert `call s:log('your string here')` wherever you need 👍